process/transactions/scr_processor.go

```diff
@@ -14,18 +14,18 @@ type scProcessor struct {
 	pubKeyConverter core.PubkeyConverter
 }
 
-// NewSCProcessor creates a new instance of smart contracts processor
-func NewSCProcessor(pubKeyConverter core.PubkeyConverter) (*scProcessor, error) {
+// NewSCResultsProcessor creates a new instance of smart contracts processor
+func NewSCResultsProcessor(pubKeyConverter core.PubkeyConverter) (*scProcessor, error) {
 	if check.IfNil(pubKeyConverter) {
 		return nil, covalent.ErrNilPubKeyConverter
 	}
 
 	return &scProcessor{pubKeyConverter: pubKeyConverter}, nil
 }
 
-// ProcessSCs converts smart contracts data to a specific structure defined by avro schema
-func (scp *scProcessor) ProcessSCs(transactions map[string]data.TransactionHandler, timeStamp uint64) []*schema.SCResult {
-	allSCRs := make([]*schema.SCResult, 0)
+// ProcessSCRs converts smart contracts data to a specific structure defined by avro schema
```

Comment:
// ProcessSCRs converts smart contract results data to a specific structure defined by avro schema

Reply:
Will change it in latest PR